### PR TITLE
DOC: Rework the examples in the docs

### PR DESCRIPTION
**Pull request by ericdill**
- Move 'Tutorial' to 'Example'.  These notebooks are really more
  examples
  than tutorials.
- Edit the introduction
- Sort the example notebooks by their parent folders in the
  scikit-beam-examples repository

### CODE REVIEW

**Comment by ericdill** [comment]
@tacaswell @danielballan Can you have a quick look at this?  No major changes. Just reworded some of the docs and are now referencing the notebooks as "Examples", not "Tutorials" as the notebooks are really more example-like than tutorial-like

**Comment by ericdill** [comment]
Or really any @scikit-beam/collaborators that are interested in our docs

File: doc/introduction.rst [hunk 1]
```diff
@@ -19,60 +21,58 @@ they play nicely with other packages from the scientific Python community.
 Further, the modular design of scikit-beam allows its components to be easily
 reused in way not envisioned by the authors.
 
+Scikit-beam is being developed at the National Synchrotron Light Source at
```
Comment (sameera2004) [inline doc/introduction.rst]: National Synchrotron Light Source II

File: doc/example/get_examples_repo.py [hunk 2]
```diff
@@ -4,11 +4,11 @@
 clone_dir = os.path.join(tempfile.gettempdir(), 'scikit-beam-examples')
 try:
     ret = subprocess.check_output(
-        ['git', 'clone', 'https://github.com/scikit-beam/scikit-beam-examples',
+        ['git', 'clone', 'https://github.com/ericdill/scikit-beam-examples',
```
Comment (sameera2004) [inline doc/example/get_examples_repo.py]: `https://github.com/ericdill/scikit-beam-examples` -->`https://github.com/scikit-beam/scikit-beam-examples`

Comment (ericdill) [inline doc/example/get_examples_repo.py]: Ah yes good catch. Thanks

**Comment by ericdill** [comment]
This is an intermittent failure. This should be good to go. @sameera2004 any other comments?

File: doc/introduction.rst [hunk 3]
```diff
@@ -19,60 +21,64 @@ they play nicely with other packages from the scientific Python community.
...
-* `Fast 2-D image conversion to Q and gridding <https://github.com/scikit-beam/scikit-beam-examples/tree/master/demos/reciprocal_space>`_
+* Differential Phase Contrast
+* MultiTau correlation
+
+    * 1-time
+    * 2-time
+    * 1-time from 2-time
+
+* Fast 2-D image conversion to Q
+* Fast gridding of 3-D point cloud into 2-D plane
```
Comment (sameera2004) [inline doc/introduction.rst]: * X-ray speckle visibility spectroscopy

Comment (ericdill) [inline doc/introduction.rst]: Good catch. Thanks

**Comment by sameera2004** [comment]
Looks good other than, one comment, why there is an error on travis for 3.4, When you fix it I will merge it.

**Comment by ericdill** [comment]
This is the failure that is intermittent and comes from floating point precision errors

**Comment by sameera2004** [comment]
Sure, Thanks Eric

**Comment by ericdill** [comment]
Great. Thanks @sameera2004 !

**Pull request merged**